Sources/Kitura/CodableRouter.swift

```diff
@@ -434,6 +434,8 @@ public struct CodableHelpers {
      *            if valid, or HTTPStatusCode.unknown otherwise
      */
     public static func httpStatusCode(from error: RequestError) -> HTTPStatusCode {
+        // ORM status code 7XX mapped to internalServerError 500
+        if error.httpCode >= 700 { return HTTPStatusCode(rawValue: 500) ?? HTTPStatusCode.unknown }
```

Comment:
Do we want to limit the replacement to strictly the 7xx range,  ie. `>= 700` and `< 800`?

Reply:
yes